coursier.bzl

```diff
@@ -330,6 +330,11 @@ Parsed artifact data: {parsed_artifact}""".format(
             )
 
             fail(error_message)
+
+        elif target_label in seen_imports:
```

Comment:
I would move this to be the first "if" condition to skip everything if we already have the target

Reply:
Done. much cleaner!